.github/workflows/build.yml

```diff
@@ -74,7 +74,7 @@ jobs:
         shell: bash
         run: |
           cd litmus-portal/graphql-server
-          docker build . -f build/Dockerfile
+          docker build . -f Dockerfile --build-arg TARGETARCH=amd64
```

Comment:
```suggestion
          docker build . -f build/Dockerfile --build-arg TARGETARCH=amd64
```

Reply:
same with other places

Reply:
I have changed the directory structure @rajdas98 and kept it common for all Dockerfiles. Earlier it was `build/Dockerfile` for some components and `Dockerfile` for others.